### DISCUSSION TITLE
ResponseStatusException should be treated as error

### DISCUSSION THREAD

**Discussion by zhenchuan9r** [open-telemetry/opentelemetry-java-instrumentation#11883] [Q&A]
I'm using javagent 1.27.
In the case of throwing ResponseStatusException from controller, the controller's span will be record as an error span.
However, this pattern is common and in most cases ResponseStatusException is just used to set response status.
The span is not "error" actually.

**Comment by trask** [comment]
hi @zhenchuan9r! starting from version 2.0, the controller spans are no longer emitted by default (you can still opt-in, but do you need them, and if not would this solve your concern)? thanks
